packages/@okta/vuepress-site/docs/guides/pwd-optional-sign-in-email/main/aspnet/integrationsummary.md

```diff
@@ -0,0 +1,5 @@
+<div class="full">
+
+![Sequence diagram illustrating the password-optional sign-in use case.](/img/pwd-optional/pwd-optional-dotnet-sign-in-summary.png)
```

Comment:
Big empty space above the "Okta email challenge flow" box.

Reply:
fixed